pkg/operator/staticpod/controller/monitoring/monitoring_resource_controller.go

```diff
@@ -35,11 +34,8 @@ type MonitoringResourceController struct {
 	targetNamespace    string
 	serviceMonitorName string
 
-	saListerSynced cache.InformerSynced
-	saLister       corelisterv1.ServiceAccountLister
-
-	clusterRoleBindingLister       rbaclisterv1.ClusterRoleBindingLister
-	clusterRoleBindingListerSynced cache.InformerSynced
+	clusterRoleBindingLister rbaclisterv1.ClusterRoleBindingLister
+	preRunCachesSynced       []cache.InformerSynced
```

Comment:
technically the PR is good. But this field name needs to be changed or at least needs a good go doc. What does "preRun" mean?